cg/services/deliver_files/delivery_file_fetcher_service/analysis_delivery_file_fetcher.py

```diff
@@ -77,7 +78,7 @@ def _get_analysis_sample_delivery_files(self, case: Case) -> list[SampleFile]:
             sample_delivery_files: list[SampleFile] = self._get_sample_files_from_case_bundle(
                 case_id=case.internal_id, sample_id=sample_id, workflow=case.data_analysis
             )
-            delivery_files.extend(sample_delivery_files)
+            delivery_files.extend(sample_delivery_files) if sample_delivery_files else None
```

Comment:
```suggestion
            delivery_files.extend(sample_delivery_files) if sample_delivery_files
```

Reply:
this will break if this is not here, since the sample_delivery_files can be none and extend will break since none is not an iterator.

Reply:
Why is the type hint not optional for the sample_delivery files?

Reply:
Also, looking at the method, it seems as if it will always be at least an empty list? So we could go with
```suggestion
            delivery_files.extend(sample_delivery_files)
```

Reply:
got it, resolve